Fix pre-commit CI

The pre-commit CI has started to fail with the following error:

```
Flake8...................................................................Failed
- hook id: flake8
- exit code: 1

bilby/bilby_mcmc/sampler.py:533:38: E226 missing whitespace around arithmetic operator
```

It's unclear to me what has caused this, but I assume it's due to a package version changing.

--- a/bilby/bilby_mcmc/sampler.py
+++ b/bilby/bilby_mcmc/sampler.py
@@ -530,7 +530,7 @@ class Bilby_MCMC(MCMCSampler):
                 else:
                     ratio = "-"
                 logger.info(
-                    f"Temp:{ii}<->{ii+1}|"  # noqa: E226
+                    f"Temp:{ii}<->{ii + 1}|"
                     f"beta={beta:0.4g}|"
                     f"hot-samp={sampler.nsamples}|"
                     f"swap={ratio}|"

--- a/bilby/bilby_mcmc/sampler.py
+++ b/bilby/bilby_mcmc/sampler.py
@@ -530,7 +530,7 @@ class Bilby_MCMC(MCMCSampler):
                 else:
                     ratio = "-"
                 logger.info(
-                    f"Temp:{ii}<->{ii+1}|"
+                    f"Temp:{ii}<->{ii+1}|"  # noqa: E226
                     f"beta={beta:0.4g}|"
                     f"hot-samp={sampler.nsamples}|"
                     f"swap={ratio}|"

